test/Microsoft.AspNet.Mvc.Core.Test/ContentResultTest.cs

```diff
@@ -131,14 +91,14 @@ public class ContentResultTest
                         new MediaTypeHeaderValue("text/foo"),
                         "κόσμε",
                         null,
-                        "text/foo; charset=utf-8",
+                        "text/foo",
```

Comment:
Generally scary how much existing tests are changing in a change mostly about refactoring existing code.